core/src/cluster_info_vote_listener.rs

```diff
@@ -434,21 +434,21 @@ impl ClusterInfoVoteListener {
             }
 
             let root_bank = bank_forks.read().unwrap().root_bank().clone();
-            if last_process_root.elapsed().as_millis() > 400 {
```

Comment:
maybe is `400` for `DEFAULT_MS_PER_SLOT`?

Reply:
yup!